src/Form.elm

```diff
@@ -233,12 +279,20 @@ update validation msg (F model) =
                         |> Maybe.map Tree.asList
                         |> Maybe.withDefault []
 
+                filterChangedFields =
+                    Set.filter (not << String.startsWith listName)
+
+                filterOriginalValue =
+                    Dict.filter (\c _ -> not <| String.startsWith listName c)
```

Comment:
educative - this can be `Dict.filter (always << not << String.startsWith listName)`.